### ISSUE TITLE
Web component: Leadspace - issue with arrows, PDF links on the buttons

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4314]
## Detailed description
On voiceover iPhone, JAWS FF

Issue: 
When we select icon type as arrow down and PDF
There is an icon near these buttons which indicates that these links downloads the file ,opens pdf 
The same information is not conveyed  for the screen reader user.

Expected:
The information conveyed visually should be available for the screen reader user as well. Anyone of the following methods can be used to make it screen reader accessible.
1. provide the information in the link text. 
2. title attribute. or 
3. as an off screen text
a generic example: 
1. feed back form opens in new window
2. products opens external link
